dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -736,6 +736,8 @@ public void sortTopLevelCategories()  throws DotDataException {
 	}
 
 	public void sortChildren(String inode)  throws DotDataException {
+        inode = SQLUtil.sanitizeParameter( inode );

```

Comment:
@jdotcms the _inode_ is a servlet param, so it could be anything, not just the UUID.